x-pack/plugins/alerts/common/builtin_action_groups.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import { ActionGroup } from './alert_type';
+
+export const ResolvedActionGroup: ActionGroup = {
+  id: 'resolved',
+  name: i18n.translate('xpack.alerts.builtinActionGroups.resolved', {
+    defaultMessage: 'Resolved',
+  }),
+};
```

Comment:
I don't think it's in scope for this PR but now that we will have multiple action groups for each alert type, would it make sense for each action group to define what it's own default message is instead of having a `defaultActionMessage` as part of the alert type definition? That way the resolved action group could have a default message and we wouldn't need to separately define a resolvedActionGroupMessage.

Reply:
Each alert type may want to specify their own Resolution default message (as the default message for any other group) - make sense to me. Because it can contains variables with params and alert data for which instance it was resolved

Reply:
thoughts on how to handle what the "default action message" is (and other fields like email subject), based on various things including action group, here: https://github.com/elastic/kibana/issues/66095#issuecomment-725203424

Reply:
@pmuellr does it make sense to leave it as it is now till the https://github.com/elastic/kibana/issues/66095#issuecomment-725203424 ?